.travis.yml

```diff
@@ -4,8 +4,7 @@ go_import_path: github.com/lyft/protoc-gen-star
 
 env:
   matrix:
-    - PROTOC_VER="3.5.1"
```

Comment:
could we please keep the other ones and just add on 3.16.0

Reply:
You may need to use https://mickey.dev/posts/go-build-tags-testing/ (for example) to mark which tests apply to the older protoc versions versus newer. But I think I really don't want to lose test coverage for the old versions, esp since it should be relatively easy to keep them. WDYT.

Reply:
Hmm, this is turning out to be tricker than I anticipated.

I need to first edit the makefiles to ensure they don't compile the proto3 optional bits, then ensure I add the build tags for the golang tests themselves..

I am testing by isolating the testing harness into `testdata/proto3_presence` for now, but let me know if you prefer a different name.